### DISCUSSION TITLE
Import csv file to IoTDB

### DISCUSSION THREAD

**Discussion by Alhonainy** [apache/iotdb#3802] [Q&A]
Hello everyone,
I am trying to import csv datasets with help of this command:
tools/import-csv.sh -h 127.0.0.1 -p 6667 -u root -pw root -f filename.csv
however, I got this error:
Meet error when insert csv because org.apache.iotdb.db.exception.metadata.IllegalPathException:

Is there anyone can guide me how to solve it?
Thank you so much

**Comment by HTHou** [comment]
Hi, the current csv tool has a lot of problems and can be considered as an unusable feature. We are working on the refactor of this part of code. 

Can you provide a sample of your csv file? So we can check if the new csv tool works properly.

**Reply by Alhonainy** [reply]
Thank you so much, I'll try to find other solutions, because I am trying to run many datasets.
